### PR TITLE
Update wrapper and deps

### PR DESCRIPTION
**Pull request by TacoTheDank**
### 1st commit
- Update gradle wrapper using git bash

### 2nd commit
- Add ndk to prevent weird compile error
- Clean up library submodule gradles
- Update Guava 28.2 -> [29.0](https://github.com/google/guava/releases)
- Update Flexbox 1.1.0 -> [1.1.1](https://github.com/google/flexbox-layout/releases)
- Update OkHttp 3.14.6 -> [3.14.7](https://square.github.io/okhttp/changelog_3x/)
- Update HTML Textview 3.4 -> [3.9](https://github.com/SufficientlySecure/html-textview/blob/master/README.md)
- Update Sentry 1.7.14 -> [1.7.30](https://github.com/getsentry/sentry-java/releases)
- Update dnsjava 3.0.0 -> [3.0.2](https://github.com/dnsjava/dnsjava/blob/master/Changelog)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -5,12 +5,6 @@ def keyStoreDefined = project.hasProperty('signingStoreLocation') &&
         project.hasProperty('signingKeyAlias') &&
         project.hasProperty('signingKeyPassword')
 
-repositories {
-    maven {
-        url 'https://jitpack.io'
-    }
-}
-
```
Comment (PerfectSlayer) [inline app/build.gradle:13]: What's the goal of moving jitpack to the root `build.gradle`?
I heard issues about jitpack repo and security. Any word about it?

File: app/build.gradle [hunk 2]
```diff
@@ -30,6 +24,8 @@ android {
         }
     }
 
+    ndkVersion "21.0.6113669"
+
```
Comment (PerfectSlayer) [inline app/build.gradle:28]: I am have a look at NDK setup. It seems I am using a `ndk.dir` in `local.properties` which is deprecated.
It seems that is the issue with failing travis build.

File: app/build.gradle [hunk 3]
```diff
@@ -101,15 +97,15 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
 
     // Miscellaneous
-    implementation 'com.google.guava:guava:28.2-android'
+    implementation 'com.google.guava:guava:29.0-android'
     implementation 'com.annimon:stream:1.2.1'
-    implementation 'com.google.android:flexbox:1.1.0'   // TODO To remove with the old home screen
-    implementation 'com.squareup.okhttp3:okhttp:3.14.6'
-    implementation 'org.sufficientlysecure:html-textview:3.4'
+    implementation 'com.google.android:flexbox:1.1.1'   // TODO To remove with the old home screen
```
Comment (PerfectSlayer) [inline app/build.gradle:102]: I need to remove it...

File: app/build.gradle [hunk 4]
```diff
@@ -101,15 +97,15 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
 
     // Miscellaneous
-    implementation 'com.google.guava:guava:28.2-android'
+    implementation 'com.google.guava:guava:29.0-android'
```
Comment (PerfectSlayer) [inline app/build.gradle:100]: I will have a loot to the changelog first as it's a new major version.

File: tcpdump/build.gradle [hunk 5]
```diff
@@ -13,8 +13,6 @@ android {
     defaultConfig {
         minSdkVersion 26
         targetSdkVersion 29
-        versionCode 1
-        versionName '1.0'
```
Comment (PerfectSlayer) [inline tcpdump/build.gradle:17]: Why would your remove version information?
Doesn't it help the overall build system?

File: app/build.gradle (hunk 1 above)
Comment (TacoTheDank) [reply app/build.gradle:13]: It just keeps everything together.

File: app/build.gradle (hunk 3 above)
Comment (TacoTheDank) [reply app/build.gradle:102]: Yeah, I know. I'm just updating it to the latest non breaking for now until you replace it.

File: app/build.gradle (hunk 4 above)
Comment (TacoTheDank) [reply app/build.gradle:100]: I looked through the changelings when making these changes. You are only using the `net` part, and nothing in net was changed.

File: tcpdump/build.gradle (hunk 5 above)
Comment (TacoTheDank) [reply tcpdump/build.gradle:17]: It's only needed for the actual application gradles. Library submodules don't need them.

File: app/build.gradle (hunk 2 above)
Comment (TacoTheDank) [reply app/build.gradle:28]: Ah ok.

**Comment by PerfectSlayer** [comment]
I merged it! Thanks 👍

File: app/build.gradle (hunk 1 above)
Comment (TacoTheDank) [reply app/build.gradle:13]: ok

**Comment by TacoTheDank** [comment]
@PerfectSlayer 

>Quick question about all gradle files, where do they come from?

Didn't notice this, sorry. The gradle files come from Gradle's own distributed builds. When you update the gradle wrapper using git bash, it automatically updates the jar and gradlew files to the specified version.

**Comment by PerfectSlayer** [comment]
Yes, I found the doc about the `gradle wrapper` command. Thanks!

**Pull request merged**